TestAssets/TestProjects/LightupLib/project.json

```diff
@@ -0,0 +1,386 @@
+{
+
+  "buildOptions": {
+    "emitEntryPoint": false
+  },
+  "dependencies": {},
+  "frameworks": {
+    "netcoreapp2.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
```

Comment:
Actually, I did test manually with lib as netstandard2.0 binary. I wanted to get the initial review going, hence so. I am making changes to get the library to build in the same manner.